apycula/gowin_pack.py

```diff
@@ -433,21 +462,31 @@ def set_iologic_attrs(db, attrs, param):
             in_attrs['CLKIMUX'] = 'ENABLE'
 
     for k, val in in_attrs.items():
-        if k not in iologic_attrids.keys():
-            print(f'XXX add {k} key handle')
+        if k not in attrids.iologic_attrids.keys():
+            print(f'XXX IOLOGIC: add {k} key handle')
```

Comment:
Another idea that I don't think is really required but could be interesting to do eventually is to use the logging module instead of printing, so we could use logging.warn here or something like that.

Reply:
That's for sure. print is very inconvenient because of the lack of structuring of messages - this XXX is a trick to be able to then search by type of messages in the output otherwise I just skipped my own debugging information :) But this is exactly a crutch.